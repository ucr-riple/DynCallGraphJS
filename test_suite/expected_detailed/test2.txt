{
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test2.js:7:1:7:9)': [
        'foo (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test2.js:1:1:3:2)'
    ],
    'foo (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test2.js:2:10:2:15)': [
        'bar (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test2.js:4:1:6:2)'
    ]
}