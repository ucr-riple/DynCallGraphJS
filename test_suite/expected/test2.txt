{
  '(test2.js:7:1:7:9)': [
    '(test2.js:1:1:3:2)'
  ],
  '(test2.js:2:10:2:15)': [
    '(test2.js:4:1:6:2)'
  ]
}